graphql-tool-idl-plugin/src/test/java/io/ballerina/graphql/idl/client/TestUtils.java

```diff
@@ -46,31 +46,22 @@ public class TestUtils {
     public static final String DISTRIBUTION_FILE_NAME = System.getProperty("ballerina.version");
     private static String balFile = "bal";
 
-    public static File[] getMatchingFiles(String project) throws IOException, InterruptedException {
-        List<String> buildArgs = new LinkedList<>();
-
-        boolean successful = executeRun(DISTRIBUTION_FILE_NAME, RESOURCE.resolve(project), buildArgs);
+    public static boolean checkModuleAvailability(String project) throws IOException, InterruptedException {
+        Process process = executeRun(DISTRIBUTION_FILE_NAME, RESOURCE.resolve(project));
         File dir = new File(new File(String.valueOf(RESOURCE.resolve(project).resolve("generated"))).toString());
-        final String id = "graphql_client";
-        File[] matchingFiles = dir.listFiles(new FileFilter() {
-            public boolean accept(File pathname) {
-                return pathname.getName().contains(id);
-            }
-        });
-        return matchingFiles;
+        return dir.exists();
```

Comment:
Shall we improve this test check for checking if there is the generated module is exist inside the `generated` directory instead of only checking the `generated` directory's existence? It would be great if we can check the source code generation inside the module. :)